### PR TITLE
sonar-l10n-zh-10.7

### PR DESCRIPTION
**Pull request by xuhuisheng**
Hi,
sonar-l10n-zh-plugin-10.7 has been released.

The main changes is support SonarQube 10.7.

The Download link and release notes: https://github.com/xuhuisheng/sonar-l10n-zh/releases/download/sonar-l10n-zh-plugin-10.7/sonar-l10n-zh-plugin-10.7.jar

The SonarCloud: https://sonarcloud.io/project/overview?id=xuhuisheng_sonar-l10n-zh

Please update the market place.

Thank you

Regards

### CODE REVIEW

**Comment by ganncamp** [comment]
While the technical release was on Friday, 10.7 won't be added to the Marketplace (the master file in this repo) until the 3rd, so I can't approve this before then. If I forget to come back to this, please ping me.

**Comment by xuhuisheng** [comment]
@ganncamp 
Oh, I didn't recognize the 10.7 didn't release, yet. Let's waiting for this weekend.

**Review by ganncamp — APPROVED** [review]

**Pull request merged**
